packages/extension-ui/src/Popup/ImportLedger.tsx

```diff
@@ -59,16 +59,20 @@ function ImportLedger ({ className }: Props): React.ReactElement {
     value
   })));
 
-  const networkOps = useRef(
-    [{
+  const networkOps = useRef([
+    {
       text: t('Select network'),
       value: ''
     },
-    ...ledgerChains.map(({ displayName, genesisHash }): NetworkOption => ({
-      text: displayName,
-      value: genesisHash[0]
-    }))]
-  );
+    ...(
+      networks
+        .filter((n) => n.hasLedgerSupport)
```

Comment:
This is still problematic, i.e. we don't pull in the data from non-substrate ss58 chains, while we may have a network